frontend/src/container/WorkspaceContent.jsx

```diff
@@ -113,6 +113,7 @@ class WorkspaceContent extends React.Component {
         props.history.push(PAGE.WORKSPACE.CONTENT_LIST(state.workspaceIdInUrl) + '?' + qs.stringify(newUrlSearch, { encode: false }))
         this.setState({ appOpenedType: false })
 
+        this.setPageTitle(props.currentWorkspace.label)
```

Comment:
I think this is the correct approach for when you have more than one place to call the custom event in a single file.
Please refactor every files calling the custom event more than one with this approach.

But rename the function name according to the new custom event name (see comment bellow)